### PR TITLE
Upgrade to Elasticsearch 7

### PR DESCRIPTION
**Pull request by pk16011990**
| Q             | A
| ------------- | ---
|Description, reason for the PR| As SSFW developer I need Elasticsearch updated to current version. Also developers need Kibana for local development.
|New feature| Yes
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| Yes
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

I didn't update UPGRADE.md

### CODE REVIEW

**Comment by pk16011990** [comment]
This PR closes #1581 and #1288

**Comment by grossmannmartin** [comment]
Hi, I rebased this pull request to the master branch as now the development version of 9.0.x release is on the master branch.

File: docker/conf/docker-compose-win.yml.dist [hunk 1]
```diff
@@ -96,6 +96,14 @@ services:
         environment:
             - discovery.type=single-node
 
+    kibana:
+        image: docker.elastic.co/kibana/kibana-oss:7.5.1
```
Comment (pesektomas) [inline docker/conf/docker-compose-win.yml.dist]: I found that current version is 7.6 (https://www.elastic.co/guide/en/kibana/current/index.html). Don't we change it yet?

File: composer.json [hunk 2]
```diff
@@ -86,7 +86,7 @@
         "doctrine/doctrine-migrations-bundle": "^1.3.0",
         "doctrine/migrations": "^1.6.0",
         "doctrine/persistence": "~1.2.0",
-        "elasticsearch/elasticsearch": "^6.0.1",
+        "elasticsearch/elasticsearch": "^7.5.0",
```
Comment (pesektomas) [inline composer.json]: Just a question: version 7.6.1 (https://packagist.org/packages/elasticsearch/elasticsearch#v7.6.1) doesn't work?

File: docker/conf/docker-compose-win.yml.dist (hunk 1 above)
Comment (pk16011990) [reply docker/conf/docker-compose-win.yml.dist]: It didn't exists 1 month ago, when I created this PR... If you want, upgrade this version...

File: composer.json (hunk 2 above)
Comment (pk16011990) [reply composer.json]: It is same "problem" like version of Elasticsearch and Kibana above. But Major and Minor version have to be same for Kibana, Elasticsearch and this package....

File: docker/conf/docker-compose-win.yml.dist (hunk 1 above)
Comment (pesektomas) [reply docker/conf/docker-compose-win.yml.dist]: I know it :) It was question for @s3tezsky :)

File: docker/conf/docker-compose.yml.dist [hunk 3]
```diff
@@ -94,6 +94,14 @@ services:
         environment:
             - discovery.type=single-node
 
+    kibana:
```
Comment (grossmannmartin) [inline docker/conf/docker-compose.yml.dist]: Shouldn't be docker/conf/docker-compose-mac.yml.dist updated too?

File: docker/conf/docker-compose.yml.dist [hunk 4]
```diff
@@ -94,6 +94,14 @@ services:
...
+        image: docker.elastic.co/kibana/kibana-oss:7.6.1
```
Comment (grossmannmartin) [inline docker/conf/docker-compose.yml.dist]: Hmm, It's impossible for me to bring up this container. it ends with error
`ERROR: manifest for docker.elastic.co/kibana/kibana-oss:7.6.1 not found: manifest unknown: manifest unknown`

Also, I don't see this tag in https://container-library.elastic.co/r/kibana/kibana-oss

File: docker/conf/docker-compose.yml.dist (hunk 3 above)
Comment (s3tezsky) [reply docker/conf/docker-compose.yml.dist]: Wow, thanks a lot for noticing!

File: docker/conf/docker-compose.yml.dist (hunk 4 above)
Comment (s3tezsky) [reply docker/conf/docker-compose.yml.dist]: That is right, there is only `7.6.0`. Now it is fixed.

**Pull request merged**